### DISCUSSION TITLE
Monai site down, any info?

### DISCUSSION THREAD

**Discussion by jovola** [Project-MONAI/MONAI#8159] [General]
The site is down for the last two days at least, not sure about earlier (oct. 16-17. 2024) any info on this?

**Comment by KumoLiu** [comment]
Hi @jovola, yes, we’re following up on this. Will fix it ASAP. Thanks.

**Comment by agokli** [comment]
Any updates related to down site please?

**Reply by aylward** [reply]
It's back!

**Comment by agokli** [comment]
ThanksSent from my iPadOn Oct 18, 2024, at 1:58 PM, Stephen R. Aylward ***@***.***> wrote:﻿
It's back!
